tokio-console/src/view/tasks.rs

```diff
@@ -149,11 +161,15 @@ impl TableList for TasksTable {
         let header_style = header_style.add_modifier(style::Modifier::BOLD);
 
         let header = Row::new(Self::HEADER.iter().enumerate().map(|(idx, &value)| {
-            let cell = Cell::from(value);
             if idx == table_list_state.selected_column {
-                cell.style(selected_style)
+                let suffix = if table_list_state.sort_descending {
+                    "▵"
+                } else {
+                    "▿"
```

Comment:
The console supports running in an ASCII-only mode, so we need to be able to switch between these Unicode characters and ASCII-only text. For other Unicode characters we use as icons, we use the `if_utf8` method on the `Styles` type to switch between Unicode and ASCII variants:
https://github.com/tokio-rs/console/blob/900a5c2bd5b610e9b939a5f824af1ac1a11267d0/tokio-console/src/view/styles.rs#L57-L63

Here, I think we should do something like this:
```suggestion
                    styles.if_utf8("▵", "+")
                } else {
                   styles.if_utf8("▿", "-") 
```
using `+` for ascending and `-` for descending in ASCII-only mode. alternatively, we could use something like `(asc.)`/`(desc.)` for ascending/descending, if we don't think a single character is descriptive enough...

note that if we use multiple characters in the ASCII-only mode, we'll also have to add that much additional length to the column width, rather than just adding a single character in every case.

Reply:
Thank you for the clear explanation. I will fix it.

Reply:
I fixed c29d6c95c7a529fb8713141ccb7630fe5240e916.